lib/engine/game/g_1880/game.rb

```diff
@@ -375,6 +426,29 @@ def train_owner(train)
         def lessee
           current_entity
         end
+
+        def round_description(name, round_number = nil)
+          round_number ||= @round.round_num
+          description = "#{name} Round "
+
+          description += @turn.to_s unless @turn.zero?
+          description += ".#{round_number}"
+
+          description += " - Train Marker at #{@train_marker.name}" if @train_marker
+
+          description.strip
+        end
+
+        def check_for_foreign_investor_connection(corp)
+          fi = @minors.find { |m| m.shares&.first&.corporation == corp }
+          return true unless fi
+
+          fi_home_token = fi.tokens.first.hex
+
+          graph = Graph.new(self, no_blocking: true)
```

Comment:
Is there a performance difference by creating the graph each time vs storing as an instance var?

Reply:
I look at the constructor. No logic going on, so what you are doing is fine.